Qiskit addons (#1882)
Qiskit/documentation: docs/guides/qiskit-addons.mdx

---
title: Qiskit addons
description: Understand the Qiskit addon tools , which help you build utility-grade quantum workflows.
---

# Qiskit addons

Qiskit addons are a collection of modular tools for building utility-scale workloads powered by Qiskit. These tools are derived from advanced research capabilities and are designed to be stackable with other tools as part of a [Qiskit pattern.](./intro-to-patterns)

## Mapping tools

> ### Multiproduct formulas
> 
> Multiproduct formulas (MPF) reduce the Trotter error of Hamiltonian dynamics through a weighted combination of several circuit executions.

Reviewer: There will be a hyphen added to the name of this addon.

CC @mrossinek, @jenglick 
```suggestion
### Multi-product formulas

Multi-product formulas (MPF) reduce the Trotter error of Hamiltonian dynamics through a weighted combination of several circuit executions.
```

- Repository: [https://github.com/Qiskit/qiskit-addon-mpf](https://github.com/Qiskit/qiskit-addon-mpf)

> - Documentation: **insert-link**

Reviewer: ```suggestion
- Documentation: https://qiskit.github.io/qiskit-addon-mpf/
```

## Optimizing tools

### Operator backpropagation

Operator backpropagation (OBP) reduces circuit depth circuit depth by trimming operations from the end at the cost of more operator measurements.

- Repository: [https://github.com/Qiskit/qiskit-addon-obp](https://github.com/Qiskit/qiskit-addon-obp)

> - Documentation: **insert-link**

Reviewer: ```suggestion
- Documentation: https://qiskit.github.io/qiskit-addon-obp/
```

### Circuit cutting

Circuit cutting reduces the depth of transpiled circuits by cutting long range gates.

- Repository: [https://github.com/Qiskit/qiskit-addon-cutting](https://github.com/Qiskit/qiskit-addon-cutting)

> - Documentation: **insert-link**

Reviewer: ```suggestion
- Documentation: https://qiskit.github.io/qiskit-addon-cutting/
```

## Postprocessing tools

### Sample-based quantum diagonalization

Sample-based quantum diagonalization (SQD) classically postprocesses noisy quantum samples to yield more accurate energy estimations.

- Repository: [https://github.com/Qiskit/qiskit-addon-sqd](https://github.com/Qiskit/qiskit-addon-sqd)

> - Documentation: **insert-link**

Reviewer: ```suggestion
- Documentation: https://qiskit.github.io/qiskit-addon-sqd/
```

# Qiskit addons

> Qiskit addons are a collection of modular tools for building utility-scale workloads powered by Qiskit. These tools are derived from advanced research capabilities and are designed to be stackable with other tools as part of a [Qiskit pattern.](intro-to-patterns)

Reviewer 2: Do you mean this? Alternatively it could be /guides/intro-to-patterns

```suggestion
Qiskit addons are a collection of modular tools for building utility-scale workloads powered by Qiskit. These tools are derived from advanced research capabilities and are designed to be stackable with other tools as part of a [Qiskit pattern.](./intro-to-patterns)
```

Author: If you're linking from a file in guides to another file in guides, why would you want /guides?

Reviewer 2: It doesn't matter which of the two styles you use. They both behave the same. So whatever you prefer
